.github/workflows/build.yml

```diff
@@ -75,6 +71,12 @@ jobs:
         reporter: dotnet-trx
       if: ${{ always() }} # Run this step even when there are test failures
 
+    - name: Check formatting
+      run: |
+        dotnet tool install -g dotnet-format --version "7.*" --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json
```

Comment:
>dotnet-format [](http://example.com/codeflow?start=31&length=13)

How is it different from the built-in `dotnet format`?

Reply:
I was having trouble with the `dotnet format` tool: when it ran online it reported issues that were not reported when I ran locally. Furthermore some of the issues were just wrong: like it suggested making a method static when the method used an instance member. I thought installing the latest version might help. But it didn't make a difference, so I can remove this line.

Reply:
Removed the dotnet-format install step.